lib/widgets/mixins/popup_menu_widget_mixin.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:fluffychat/widgets/fluffy_chat_app.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:linagora_design_flutter/colors/linagora_sys_colors.dart';
+
+typedef OnTapIconButtonCallbackAction = void Function();
```

Comment:
why you define type but you don't reuse it

Reply:
done